.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -1,27 +1,69 @@
-name: Feature Request
-description: Request a new feature for FluentHub
+name: ✨ Feature Request
+description: Request a new feature to make things better!
 labels: [feature request]
+title: "Feature: "
 body:
-  - type: checkboxes
+  - type: markdown
     attributes:
-      label: Before you start...
-      options:
-        - label: Have you checked if a similar feature request has already been reported?
-          required: true
+      value: |
+        ##### ⏱️ Before you start...
+        - Have you checked whether or not a similar feature request has already been reported?
   - type: textarea
     attributes:
-      label: Description
+      label: 📄 Description
       description: A clear and concise description of what your idea is. Include things like possible use cases, drawbacks, etc.
     validations:
       required: true
   - type: textarea
     attributes:
-      label: Alternative solutions
+      label: 🗃️ Alternative solutions
       description: Describe more ways this idea could be implemented.
     validations:
       required: false
   - type: textarea
+    id: api-changes
     attributes:
-      label: Assets
-      description: |
-        A list of assets (screenshots, mockups) relevant to this feature request.
+      label: ⚡ API Changes
```

Comment:
It'd be better to be 'Requirements'.
Having issuer think API spec is a bit harsh.